test_runner/regress/test_sni_router.py

```diff
@@ -60,8 +63,12 @@ def start(self) -> "PgSniRouter":
             *["--destination", self.destination],
         ]
 
-        self._popen = subprocess.Popen(args)
+        router_log_path = self.test_output_dir / "pg_sni_router.log"
+        router_log = open(router_log_path, "w")
```

Comment:
this will always truncate, probably not what we really want but I don't have a grasp on the test_output dir deletion so this is probably good enough.